department-of-veterans-affairs/abd-vro
# 1821: Created Max CFI API application.

Added MAX CFI API to API Gateway. Added integration test for API Gateway. Added configurations to secrel/config.yml, delete-published-dev-images.yml, and delete-published-images.yml



## What was the problem?
Team EE needs an API to retrieve the max rating for a list of diagnostic codes. 

Associated tickets or Slack threads:
- #1821

## How does this fix it?[^1]

EE Max CFI API application created with a single post endpoint. 

## How to test this PR
- refer to README in the `domain-ee/README.md` folder for testing instructions
- successful run: [continuous-integration](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/5672041601/job/15370423901)

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
